R/coord-radial.R

```diff
@@ -40,9 +41,21 @@
 #' ggplot(mtcars, aes(disp, mpg)) +
 #'   geom_point() +
 #'   coord_radial(start = -0.4 * pi, end = 0.4 * pi, inner.radius = 0.3)
+#' 
+#' # Similar with coord_cartesian(), you can set limtis, but note the `clip` 
+#' # area is not the same with the circle track area.
```

Comment:
```suggestion
#' # Similar with coord_cartesian(), you can set limits, but note the `clip` 
#' # area is not the same with the circle track area.
```
The clipping not applying properly, is that something that should be fixed?